kafka-ui-react-app/src/components/Dashboard/__test__/Dashboard.spec.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import { shallow } from 'enzyme';
 import Dashboard from 'components/Dashboard/Dashboard';
+import { render } from 'lib/testHelpers';
 
-const component = shallow(<Dashboard />);
+const component = render(<Dashboard />);
 
 describe('Dashboard', () => {
   it('renders ClustersWidget', () => {
-    expect(component.exists('Connect(ClustersWidget)')).toBe(true);
+    expect(component.queryAllByText('clusters').length).toBeTruthy();
```

Comment:
why use query if the text is in the Document use getBy => and if searching for specific length make that equal to the length , if it is a single element do it with toBeinDocument

Reply:
done